proxies/live/apiproxy/targets/ig3.xml

```diff
@@ -22,7 +22,7 @@
       </Step>
       <Step>
         <Name>AssignMessage.SetAccessModeUserRestricted</Name>
-        <Condition>(scope JavaRegex "(.+\ urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service\ .+|^urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service\ .+|.+\ urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service$|^urn:nhsd:apim:user-nhs-id:aal3:personal-demographics-service$|.+\ personal-demographics-service:USER-RESTRICTED\ .+|^personal-demographics-service:USER-RESTRICTED\ .+|.+\ personal-demographics-service:USER-RESTRICTED$|^personal-demographics-service:USER-RESTRICTED$)") or ((scope JavaRegex "(.+\ urn:nhsd:apim:app:level3:personal-demographics-service\ .+|^urn:nhsd:apim:app:level3:personal-demographics-service\ .+|.+\ urn:nhsd:apim:app:level3:personal-demographics-service$|^urn:nhsd:apim:app:level3:personal-demographics-service$)" and apim-app-flow-vars.pds.app-restricted.update Is "true")</Condition>
+        <Condition>(scope JavaRegex "(.+\ urn:nhsd:apim:user-nhs-cis2:aal3:personal-demographics-service\ .+|^urn:nhsd:apim:user-nhs-cis2:aal3:personal-demographics-service\ .+|.+\ urn:nhsd:apim:user-nhs-cis2:aal3:personal-demographics-service$|^urn:nhsd:apim:user-nhs-cis2:aal3:personal-demographics-service$|.+\ personal-demographics-service:USER-RESTRICTED\ .+|^personal-demographics-service:USER-RESTRICTED\ .+|.+\ personal-demographics-service:USER-RESTRICTED$|^personal-demographics-service:USER-RESTRICTED$)") or ((scope JavaRegex "(.+\ urn:nhsd:apim:app:level3:personal-demographics-service\ .+|^urn:nhsd:apim:app:level3:personal-demographics-service\ .+|.+\ urn:nhsd:apim:app:level3:personal-demographics-service$|^urn:nhsd:apim:app:level3:personal-demographics-service$)" and apim-app-flow-vars.pds.app-restricted.update Is "true")</Condition>
```

Comment:
After discussing this on Wednesday morning, do we know if we need to inform PDS FHIR API users about the change in scopes?

Reply:
After discussing this on Wednesday morning, do we know if we need to inform PDS FHIR API users about the change in scopes?